mopidy_alsamixer/mixer.py

```diff
@@ -70,13 +74,20 @@ def get_volume(self):
         if not channels:
             return None
         elif channels.count(channels[0]) == len(channels):
-            return int(channels[0])
+            unadjusted_volume = channels[0]
+            if self.logarithmic_volume:
+                unadjusted_volume = 100.0 * math.pow(unadjusted_volume / 100.0, 3.0)
```

Comment:
Line 89 is actually the inverse of the function on line 79. Line 79 undoes the conversion performed on line 89.

I think using those conversion functions seems like a great idea. I'll make that change and support db (logarithmic) as well. Reading those docs confirms what I implemented was cubic. I mean, there was a `math.pow(x, 3)` in there... so it was pretty obvious.